docs/source/python/index.rst

```diff
@@ -47,3 +47,4 @@ files into Arrow structures.
    api
    development
    getting_involved
+   benchmarks
```

Comment:
Perhaps we may want to group "development", "getting involved" and "benchmarks" in a subchapter.

Reply:
Is there a simple way to do this in a `toctree` or should we start a subfolder `contributing`?